docs/examples/movie-reviews-graph-rag.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>I think we can omit this code block</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datastax/graph-rag/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>